MH-13558, Paella Track Filter

lkiesow opened the pull request:
To resolve the discussion about track filtering from pull request #896,
this patch adds a configurable track filter.

lkiesow committed:
MH-13558, Paella Track Filter

To resolve the discussion about track filtering from pull request #896,
this patch adds a configurable track filter.

miesgre commented on etc/ui-config/mh_default_org/paella/config.json:
```diff
@@ -119,6 +119,9 @@
     "//**** For a list of available plugins and configuration, go to": "https://github.com/polimediaupv/paella/blob/master/doc/plugins.md",
     "list": {
+      "es.upv.paella.opencast.loader": {
+        "flavor": "*/*"
```
Maybe it's better to have a list of accepted flavors

lkiesow replied:
done.

lkiesow committed:
MH-13558, Allow Multiple Filters

This patch allows multiple flavors for track filtering in the Paella
player to be specified.

miesgre commented on etc/ui-config/mh_default_org/paella/config.json:
```diff
@@ -119,6 +119,11 @@
     "list": {
+      "es.upv.paella.opencast.loader": {
+        "flavors": [
+          ["*", "*"]
```
Why Do you choose this notation instead of "*/*" to be consistent with other workflow configuration files?
I think it is more readable 
```
flavors:["*/*"]
```
than
```
flavors:[["*", "*"]]
```

What do you think?

lkiesow replied:
It's essentially the same and the array variant is less to parse.

staubesv commented:
@lkiesow Please don't forget to adjust the upgrade documentation ;-)

miesgre reviewed (approved):
It's fine to me
